packages/mdc-select/component.ts

```diff
@@ -53,17 +53,13 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
 
   ripple!: MDCRipple | null;
 
-  private menu_!: MDCMenu | null; // assigned in enhancedSelectSetup_()
+  private menu_!: MDCMenu; // assigned in enhancedSelectSetup_()
```

Comment:
nit: Remove/update `enhancedSelectSetup_` code comment.

Same below.

Reply:
Renamed these in a future PR.